website/source/assets/stylesheets/_variables.scss

```diff
@@ -51,8 +51,7 @@ $link-hover-color:      darken($link-color, 15%);
 // Typography
 // -------------------------
 $font-family-open-sans: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$font-family-blanc:     'Blanc-Light', 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$font-family-blanc-reg: 'Blanc-Regular', 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-klavika:   'klavika-web', 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
@jasoncostello if we are making sweeping changes across all the projects, we can put some of this in middleman-hashicorp too